addon/services/pollboy.js

```diff
@@ -13,11 +13,15 @@ export default Service.extend({
   init () {
     this._super()
     this._onVisibilityChangeHandler = this.onVisibilityChange.bind(this)
-    document.addEventListener('visibilitychange', this._onVisibilityChangeHandler, false)
+    if (typeof document !== 'undefined') {
```

Comment:
Can `if (document)` be used instead?  (not sure if there's something special about the FastBoot environment that it can't be)

Reply:
It's really up to you.  This seems to be the preferred method elsewhere (including the fast boot docs) though `if (document)` seems fine to me.  Would you prefer that I change it?

Reply:
Nah, if it's in the FastBoot docs this way I am fine with it.  We have not given any attention to FastBoot specifically so am fine following your direction.